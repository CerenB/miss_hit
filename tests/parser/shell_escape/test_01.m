% (c) Copyright 2019 Florian Schanda

function test_01()
  !echo foo bar
end
