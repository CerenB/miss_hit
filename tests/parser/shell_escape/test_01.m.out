----------------------------------------------------------------------
--  Parse tree for test_01.m
Root: Function_File
  Functions: Function_Definition
    Signature: Function_Signature
      Name: Identifier <test_01>
    Body: Sequence_Of_Statements
      Statements: Naked_Expression_Statement
        Expression: Escape form Function_Call
          Arguments: Char_Array_Literal 'echo foo bar'
----------------------------------------------------------------------
In test_01.m, line 3
| function test_01()
|                 ^ info: this token is not linked to the ast
In test_01.m, line 3
| function test_01()
|                  ^ info: this token is not linked to the ast
In test_01.m, line 5
| end
| ^^^ info: this token is not linked to the ast
MISS_HIT Summary: 1 file(s) analysed, everything seemes fine
